spec/win32/unit/certstore_spec.rb

```diff
@@ -1,58 +0,0 @@
-#
-# Author:: Nimisha Sharad (<nimisha.sharad@msystechnologies.com>)
-# Copyright:: Copyright (c) 2017 Chef Software, Inc.
-# License:: Apache License, Version 2.0
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-require 'spec_helper'
-
-describe Win32::Certstore do
-  class Certstore
-    include Win32::Certstore
-  end
-
-  before do
-    @certstore = Certstore.new
-  end
-
-  it 'has a version number' do
-    expect(Win32::Certstore::VERSION).not_to be nil
-  end
-
-  describe "#open" do
```

Comment:
Please update the specs for `certstore` as per the code changes. They shouldn't be commented out.